numpy/fft/tests/test_helper.py

```diff
@@ -36,10 +32,112 @@ def test_axes_keyword(self):
         shifted = [[-1, -3, -2], [2, 0, 1], [-4, 3, 4]]
         assert_array_almost_equal(fft.fftshift(freqs, axes=(0, 1)), shifted)
         assert_array_almost_equal(fft.fftshift(freqs, axes=0),
-                fft.fftshift(freqs, axes=(0,)))
+                                  fft.fftshift(freqs, axes=(0,)))
         assert_array_almost_equal(fft.ifftshift(shifted, axes=(0, 1)), freqs)
         assert_array_almost_equal(fft.ifftshift(shifted, axes=0),
-                fft.ifftshift(shifted, axes=(0,)))
+                                  fft.ifftshift(shifted, axes=(0,)))
+
+        assert_array_almost_equal(fft.fftshift(freqs), shifted)
+        assert_array_almost_equal(fft.ifftshift(shifted), freqs)
+
+    def test_uneven_dims(self):
+        """ Test input with uneven dimension lengths """
+        freqs = [
+            [0, 1],
+            [2, 3],
+            [4, 5],
+            [6, 7]
```

Comment:
Might be good to have one of the dimensions be an odd number, like it was in your previous test. Maybe a 3x4 array.

Reply:
I used an 3x2 array as it behaves the same as 3x4 but is easier to read.